client/build.gradle

```diff
@@ -44,9 +44,6 @@ dependencies {
   signature "org.codehaus.mojo.signature:java17:1.0@signature"
 }
 
-compileJava.options.compilerArgs << '-target' << '1.7' << '-source' << '1.7' << '-Xlint:all,-path,-serial,-options'
```

Comment:
This should also work with forbiddenapis, because it inherits targetCompatibility. Just do "gradle --info forbiddenApis" and check whats printed out while loading signatures files.